pyaedt/modeler/cad/Primitives3D.py

```diff
@@ -1659,7 +1660,8 @@ def insert_layout_component(
             Map the layout parameters in the target HFSS design. The default is ``False``.
```

Comment:
```suggestion
            Whether to map the layout parameters in the target HFSS design. The default is ``False``.
```
Edit to follow guidelines--just because I could see this old inconsistency in style.